libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json

```diff
@@ -128,7 +128,7 @@
         "limit": 0
     },
     "options.menu.schedule.email": {
-        "value": "Schedule exports",
+        "value": "Schedule export",
```

Comment:
This will also override top bar menu item translation and that should be in plural. Rather create a new singular item for your menu